core/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -108,6 +108,23 @@ class FormatOps(val tree: Tree, val initStyle: ScalafmtConfig) {
     (packages.result(), imports.result(), arguments.toMap, optional)
   }
 
+  object CtorModifier {
+    def is(tree: Tree): Boolean =
+      tree.is[meta.Mod.Private] || tree.is[meta.Mod.Protected]
+    def is(tok: Token): Boolean = tok.is[KwPrivate] || tok.is[KwProtected]
+    def unapply(tok: Token): Boolean = is(tok)
+  }
+
+  object LeftParenOrBracket {
```

Comment:
Can we move this to https://github.com/scalameta/scalafmt/blob/235909ffdcbefe18457878a8c95c6c93ff288ed9/core/src/main/scala/org/scalafmt/util/TokenClasses.scala ?